src/components/payments/_global/PaymentStatusError.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div class="dcww-payments">
+
+    <header class="dcww-payments-header">
+        <a class="dcww-payments__link dcww-payments__link--image" href="https://www.dwrcymru.com/"><img src="../../../assets/dcww.png" alt="logo" class="dcww-logo"></a>
+        <div class="dcww-payments-header__links">
+            <a class="dcww-payments__link" href="">Cymraeg</a>
+        </div>
+    </header>
+
+    <main class="dcww-payments-form">
+        <div class="dcww-payments-form__container">
+            <div class="dcww-payments-status">
+                <img src="../../../assets/caution.svg" class="dcww-payments-status__icon--large">
```

Comment:
all of these svgs I would consider doing inline as eckoh are just going to have the plain html right? None of the other files

Reply:
We are sending them an assets bundle